website/content/docs/operations/metrics-reference.mdx

```diff
@@ -202,6 +202,7 @@ task driver; not all task drivers can provide all metrics.
 | `nomad.client.allocs.cpu.user`                | Total CPU resources consumed by the task in the user space        | Percentage  | Gauge   | alloc_id, host, job, namespace, task, task_group |
 | `nomad.client.allocs.failed`                  | Number of failed allocations                                      | Integer     | Counter | alloc_id, host, job, namespace, task, task_group |
 | `nomad.client.allocs.memory.allocated`        | Amount of memory allocated by the task                            | Bytes       | Gauge   | alloc_id, host, job, namespace, task, task_group |
+| `nomad.client.allocs.memory.max_allocated`    | Maximum amount of oversubscription memory allocated by the task   | Bytes       | Gauge   | alloc_id, host, job, namespace, task, task_group |
```

Comment:
Open to suggestions for this name 😄

Reply:
Ops, good catch. In of my iterations this was called `nomad.client.allocs.memory.allocated_max` 😅